src/test/java/org/folio/rest/camunda/utility/FormatUtilityTest.java

```diff
@@ -0,0 +1,120 @@
+package org.folio.rest.camunda.utility;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class FormatUtilityTest {
+
+  static class TestData {
```

Comment:
I suggest moving the static class to the bottom of the class to get it out of the way.
I also suggest using the Javadocs to document the purpose of this class.

As an example, here is how I've done this:
- https://github.com/folio-org/mod-workflow/blob/d6e4905946c22ac74915246b482ad18066e426a5/service/src/test/java/org/folio/rest/workflow/service/WorkflowImportServiceTest.java#L358

I also suggest the same for the `sqlStream()` method.
It would be great to have consistent testing practices across our projects.

see: https://github.com/folio-org/mod-workflow/blob/d6e4905946c22ac74915246b482ad18066e426a5/components/src/test/java/org/folio/rest/workflow/utility/CompressFileMagicTest.java#L180

Reply:
I have moved the inner classes to the bottom.

Reply:
Moved all static methods for parameterized tests to the bottom of the file.